specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2020-10-01/ComputeOperationResults.json

```diff
@@ -0,0 +1,193 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Compute operation results.",
+    "title": "FabricAdminClient",
+    "version": "2020-10-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Fabric.Admin/fabricLocations/{location}/computeOperationResults/{operation}": {
+      "get": {
+        "x-ms-examples": {
+          "Returns the status of the operation.": {
+            "$ref": "./examples/ComputeOperationResult/Get.json"
+          }
+        },
+        "tags": [
+          "ComputeOperationResults"
+        ],
+        "description": "Returns the status of a compute operation.",
+        "operationId": "ComputeOperationResults_Get",
+        "parameters": [
+          {
+            "$ref": "../2016-05-01/Fabric.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../2016-05-01/Fabric.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "../2016-05-01/Fabric.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "#/parameters/OperationParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ComputeOperationResult"
+            }
+          },
+          "404": {
+            "description": "NOT FOUND"
+          }
```

Comment:
in general, 404 is regarded as an error.  we use  "default" to cover all error response code.

please remove the 404 block and add a default block

Reply:
Fixed